FiveDChessDataInterface/Types/ChessBoard.cs

```diff
@@ -49,7 +49,7 @@ public ChessBoard(ChessBoardMemory mem, int width, int height)
                             i += 1;
                             x += 1;
                         }
-                        pieces += $"{i}";
+                        pieces += $"{i}"+p.FENSymbol();
```

Comment:
`p` is still the the same as when p.isEmpty was true, and this doesn't consider the case where `x==width` and there isn't another piece on this row to include. One solution would be:
```
pieces += $"{i}";
if(x<width) pieces += Pieces[x*height+y].FENSymbol();
```